graspy/simulations/rdpg_corr.py

```diff
@@ -0,0 +1,114 @@
+import numpy as np
+from graspy.simulations import p_from_latent, sample_edges_corr
+
+
+def rdpg_corr(X, Y, r, rescale, directed, loops):
```

Comment:
@SHAAAAN some of these should be kwargs, like normal

Reply:
I don't understand what do you mean here, do you mean adding the default values?

Reply:
yes, these are called kwargs (keyword arguments)